lib/util/error.js

```diff
@@ -871,13 +871,7 @@ shaka.util.Error.Code = {
    */
   'STORE_ALREADY_IN_PROGRESS': 9006,
 
-  /**
-   * The specified manifest is encrypted but does not specify any init data.
-   * Without init data specified in the manifest, the content will not be
-   * playable offline.
-   * <br> error.data[0] is the URI.
-   */
-  'NO_INIT_DATA_FOR_OFFLINE': 9007,
+  // RETIRED: 'NO_INIT_DATA_FOR_OFFLINE': 9007,
```

Comment:
Thank you for maintaining this!  :+1:

Reply:
Since it was no longer necessary I thought this was the right thing to do